tests/integration/targets/ios_vrf_interfaces/tests/common/empty_config.yaml

```diff
@@ -0,0 +1,50 @@
+---
+- ansible.builtin.debug:
+    msg: START ios_vrf_interfaces empty_config integration tests on connection={{ ansible_connection }}
+
+- name: Merged with empty configuration should give appropriate error message
+  register: result
+  ignore_errors: true
+  cisco.ios.ios_vrf_interfaces:
+    config:
+    state: merged
+
+- ansible.builtin.assert:
+    that:
+      - result.msg == 'value of config parameter must not be empty for state merged'
+
+- name: Replaced with empty configuration should give appropriate error message
```

Comment:
can we also include one task for state: overridden

Reply:
done